TheKoans/AboutStrings.cs

```diff
@@ -17,22 +17,22 @@ public class AboutStrings : KoanHelper
         public void DoubleQuotedStringsAreStrings()
         {
             var str = "Hello, World";
-            Assert.AreEqual(typeof(FILL_ME_IN), str.GetType(), "What did Arial say to Serif? - Sorry, you're not my Type.");
+            Assert.AreEqual(typeof(string), str.GetType(), "What did Arial say to Serif? - Sorry, you're not my Type.");
         }
 
         [TestMethod]
         public void SingleQuotedStringsAreNotStrings()
         {
             // Note the use of single quotes below.  They're not double quotes, which are definitely used for strings.
             var str = 'H';
-            Assert.AreEqual(typeof(FILL_ME_IN), str.GetType(), "A single tree does not a forest make.");
+            Assert.AreEqual(typeof(Char), str.GetType(), "A single tree does not a forest make.");
```

Comment:
Not a huge deal, but we use lower case classes for what we consider "primititve" data types.  Either works because in C# they are aliases, but it's a style convention.  Java uses upper case, in C# we use lower case.  (i.e. char, string, etc)